.evg.yml

```diff
@@ -47,6 +47,16 @@ buildvariants:
       - name: "test-unit"
       - name: "test-integration"
 
+  - name: ubuntu2204-arm64-jdk-8
```

Comment:
The distro we'd be trying this on closely matches what we actually would use in production, `amazon2-arm64-graviton3`. What do you think about swapping to this?

Reply:
Sure, I started up an `amazon2-arm64-graviton3` host to test it out.  I'll likely add another check for this distro with different links.